config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -399,10 +445,11 @@ presubmits:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210917-ee1e7c845b-1.21
         command:
         - runner.sh
-        - bash
         args:
+        - bash
         - -c
         - |
+          iptables -t mangle -A POSTROUTING -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
```

Comment:
we can remove this for runs of `apidiff` as well

Reply:
Removed it from the two invocations.